### ISSUE TITLE
1.3.17 lost FQN

### ISSUE THREAD

**Issue by lsoft** [zompinc/sync-method-generator#55]
```C#
        [Zomp.SyncMethodGenerator.CreateSyncVersion]
        public virtual async Task<bool> SomeMethodAsync(
            object a
            )
        {
            await Task.Delay(0).ConfigureAwait(false);

            return a is not DBNull;
        }
```

the result:

![image](https://github.com/zompinc/sync-method-generator/assets/5988558/ee4a1cb0-da86-4bf9-a063-baa6b56260f2)

**Comment by lsoft** [comment]
`out` parameters is not processed too:

```C#
        [Zomp.SyncMethodGenerator.CreateSyncVersion]
        public async Task MyMethodAsync()
        {
            Dictionary<int, XmlArrayAttribute> q = null;
            q.TryGetValue(0, out XmlArrayAttribute a);
            await Task.Delay(0);
```

![image](https://github.com/zompinc/sync-method-generator/assets/5988558/37c66d55-bb06-4644-9d3a-d50998030d5c)

**Comment by virzak** [comment]
Available in 1.3.18
